includes/class-newspack-popups-importer.php

```diff
@@ -228,9 +228,17 @@ private function process_prompts() {
 		$prompts = $this->pre_process_prompts_terms( $prompts );
 
 		foreach ( $prompts as $prompt ) {
+			$prompt_content = $prompt['content'];
+			$user_inputs    = isset( $prompt['user_inputs'] ) ? $prompt['user_inputs'] : [];
+
+			foreach ( $user_inputs as $user_input ) {
+				$value          = ! empty( $user_input['value'] ) ? $user_input['value'] : $user_input['default'];
+				$prompt_content = str_replace( '{{' . $user_input['name'] . '}}', $value, $prompt_content );
```

Comment:
shouldn't we crop the string if manx_length is set?

Reply:
Yes, but I was planning to leave this up to the React (UI) part of things.

Reply:
Makes sense. But since you can also run the importer via CLI, I think we should make sure the back-end also handles it

Reply:
Perhaps better to do it in both places, though. Added in 269e0fc